core/services/chainlink/example_test.go

```diff
@@ -1,39 +0,0 @@
-package chainlink_test
```

Comment:
These cases are redundant and Examples are particularly helpful here anyways, so might as well avoid the set/clear env mess.